Quest helper tripped assert regarding rune pouch contents

```
2025-07-22 14:00:34 PDT [Client] ERROR injected-client - Client error
java.lang.AssertionError: null
	at com.questhelper.managers.QuestContainerManager.updateInventory(QuestContainerManager.java:104)
	at com.questhelper.QuestHelperPlugin.onItemContainerChanged(QuestHelperPlugin.java:263)
	at net.runelite.client.eventbus.EventBus$Subscriber.invoke(EventBus.java:70)
	at net.runelite.client.eventbus.EventBus.post(EventBus.java:223)
	at net.runelite.client.callback.Hooks.post(Hooks.java:205)
	at client.ml(client.java:24046)
	at client.pj(client.java:13507)
	at client.at(client.java:12035)
	at bo.bl(bo.java:467)
	at bo.vy(bo.java)
	at bo.run(bo.java:60859)
	at java.base/java.lang.Thread.run(Thread.java:840)
```

This happened when withdrawing my (empty) rune pouch from the bank.

--- a/src/main/java/com/questhelper/managers/ItemAndLastUpdated.java
+++ b/src/main/java/com/questhelper/managers/ItemAndLastUpdated.java
@@ -30,6 +30,7 @@ import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
 import net.runelite.api.Item;
 
+import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import java.util.concurrent.Callable;
 
@@ -42,7 +43,7 @@ public class ItemAndLastUpdated
     // last game tick item container was changed
     @Getter
     private int lastUpdated = -1;
-    private Item[] items;
+    private Item[] items = new Item[0];
 
     @Setter
     private Callable<Item[]> specialMethodToObtainItems;
@@ -65,7 +66,7 @@ public class ItemAndLastUpdated
      *
      * @return an {@link Item}[] of items currently thought to be in the container.
      */
-    public @Nullable Item[] getItems()
+    public @Nonnull Item[] getItems()
     {
         if (specialMethodToObtainItems != null)
         {

--- a/src/main/java/com/questhelper/managers/QuestContainerManager.java
+++ b/src/main/java/com/questhelper/managers/QuestContainerManager.java
@@ -101,9 +101,9 @@ public class QuestContainerManager
         if (result)
         {
             Item[] runesInPouch = QuestContainerManager.getRunePouchData().getItems();
-            assert runesInPouch != null;
             for (Item runePouchItem : runesInPouch)
             {
+                if (runePouchItem == null) continue;
                 inventoryMap.computeIfPresent(runePouchItem.getId(), (currentVal, existingItem) -> new Item(currentVal, existingItem.getQuantity() + runePouchItem.getQuantity()));
                 inventoryMap.putIfAbsent(runePouchItem.getId(), runePouchItem);
             }

